src/main/resources/db/migration/v0.34.0.6__gene_expression_annotation.sql

```diff
@@ -0,0 +1,69 @@
+CREATE SEQUENCE public.expressionannotation_seq         START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE TABLE expressionannotation (
+	id bigint CONSTRAINT expressionannotation_pkey PRIMARY KEY,
+	relation_id bigint,
+	whenexpressedstagename VARCHAR(2000),
+	whereexpressedstatement VARCHAR(2000)
+);
+
+ALTER TABLE expressionannotation
+	ADD CONSTRAINT  expressionannotation_relation_id_fk
+		FOREIGN KEY (relation_id) REFERENCES vocabularyterm (id);
+
+CREATE INDEX expressionannotation_whenexpressedstagename_index ON expressionannotation USING btree (whenexpressedstagename);
+CREATE INDEX expressionannotation_whereexpressedstatement_index ON expressionannotation USING btree (whereexpressedstatement);
+
+CREATE SEQUENCE public.geneexpressionannotation_seq         START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
```

Comment:
The INCREMENT here needs to be set to 50 as that is what hibernate wants.

Reply:
Changed to 50